manifest.yaml

```diff
@@ -111,6 +112,61 @@ postprocess:
      echo '# RHCOS-only: we follow the Fedora/upstream default' >> /usr/lib/systemd/system/basic.target
      echo 'Wants=tmp.mount' >> /usr/lib/systemd/system/basic.target
 
+     # Ensure that /etc/issue.d exists for console-login-helper-messages
+     # This can be removed once we rebase to RHEL 9
+     install -d -m 0755 /etc/issue.d
+
+     # Tweak /usr/lib/os-release
+     OCP_RELEASE="4.11"
+     (
+     . /etc/os-release
+     cat > /usr/lib/os-release <<EOF
+     NAME="Red Hat Enterprise Linux CoreOS"
+     ID="rhcos"
+     ID_LIKE="rhel fedora"
+     VERSION="${OSTREE_VERSION}"
+     VERSION_ID="${OCP_RELEASE}"
+     PLATFORM_ID="${PLATFORM_ID}"
+     PRETTY_NAME="Red Hat Enterprise Linux CoreOS ${OSTREE_VERSION} (Ootpa)"
+     ANSI_COLOR="${ANSI_COLOR}"
+     CPE_NAME="cpe:/o:redhat:enterprise_linux:8::coreos"
+     HOME_URL="https://www.redhat.com/"
+     DOCUMENTATION_URL="https://docs.openshift.com/container-platform/${OCP_RELEASE}/"
+     BUG_REPORT_URL="https://bugzilla.redhat.com/"
+     REDHAT_BUGZILLA_PRODUCT="OpenShift Container Platform"
+     REDHAT_BUGZILLA_PRODUCT_VERSION="${OCP_RELEASE}"
+     REDHAT_SUPPORT_PRODUCT="OpenShift Container Platform"
+     REDHAT_SUPPORT_PRODUCT_VERSION="${OCP_RELEASE}"
+     OPENSHIFT_VERSION="${OCP_RELEASE}"
+     RHEL_VERSION="${VERSION_ID}"
+     OSTREE_VERSION="${OSTREE_VERSION}"
+     EOF
+     )
+
+     # Tweak /etc/system-release, /etc/system-release-cpe & /etc/redhat-release
+     (
+     . /etc/os-release
+     cat > /etc/system-release <<EOF
+     ${NAME} release ${VERSION_ID}
+     EOF
+     cat > /etc/system-release <<EOF
+     ${NAME} release ${VERSION_ID}
+     EOF
+     cat > /etc/system-release-cpe <<EOF
+     ${CPE_NAME}
+     EOF
+     )
+
+     # Tweak /etc/issue & /etc/issue.net
+     rm /etc/issue
+     ln -s /usr/lib/issue /etc/issue
+     cat > /usr/lib/issue <<EOF
+     \S \S{VERSION_ID}
+     EOF
+     cat > /usr/lib/issue.net <<EOF
+     \S \S{VERSION_ID}
+     EOF
```

Comment:
This is just a small suggestion to put writing content behind creating link.

     rm /etc/issue
     
     cat > /usr/lib/issue <<EOF
     \S \S{VERSION_ID}
     EOF
     ln -s /usr/lib/issue /etc/issue
     ln -s /usr/lib/issue /etc/issue.net